README.rst

```diff
@@ -1196,6 +1197,44 @@ being set to ``True`` (which is the default).
 You can turn this off if you do not use the geographic features
 of OpenWISP.
 
+``OPENWISP_MONITORING_DASHBOARD_TRAFFIC_CHART``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+--------------------------+
+| **type**:    | ``dict``                 |
++--------------+--------------------------+
+| **default**: | ``{'__all__': ['wan']}`` |
```

Comment:
I realized this default value won't work on most systems because openwrt doens't call the interfaces wan but eth1 or eth0.1, so maybe a better default value is a list containing all these values (wan, eth1, eth0.1).

Reply:
The default value here needs to be updated.